taxonomy: small fixes on various taxonomies (obf brands & labels, off ingredients, opf categories/unspsc, unused brands categories, additives, nutrients, other-nutrients)

teolemon opened the pull request:
### What
taxonomy: small fixes on various taxonomies (obf brands & labels, off ingredients, opf categories/unspsc, unused brands categories, additives, nutrients, other-nutrients)

teolemon committed:
small fixes on various taxos

teolemon committed:
wikidata items on unused taxos

Commit:
chore: Linting changes

teolemon committed:
Merge branch 'main' into small-things

Freso commented on taxonomies/unused/categories.wip.txt:
```diff
@@ -4030,21 +4030,21 @@ fr: Beaumes de Venise
 protected_name_file_number:en: PDO-FR-A0737
 protected_name_type:en: pdo
-#wikidata:en:
+#wikidata:en:Q2922358
```
```suggestion
wikidata:en:Q2922358
```

Freso commented on taxonomies/other_nutritional_substances.txt:
```diff
@@ -90,7 +92,7 @@ vegetarian:en: maybe
 
-# description:en:L-CARNITINE is the left isomer of carnitine, a quaternary ammonium compound involved in metabolism in most mammals, plants and some bacteria. The body synthesizes enough carnitine from lysine side chains to keep up with the needs of energy production in the body as carnitine acts as a transporter of long-chain fatty acids into the mitochondria to be oxidized and produce energy. F ...
+
 
```
```suggestion
```

Freso commented on taxonomies/other_nutritional_substances.txt:
```diff
@@ -276,14 +275,16 @@ tr: Kolin
 wikipedia:en: https://en.wikipedia.org/wiki/Choline
+# comment:en:Related to additive E1001 and vitamin B4
+
 
```
```suggestion
```

Freso commented on taxonomies/other_nutritional_substances.txt:
```diff
@@ -312,9 +313,11 @@ sr: Holin hlorid
 wikidata:en: Q2964153
 wikipedia:en: https://en.wikipedia.org/wiki/Choline_chloride
-# ingredient/choline-chloride has 18 products in 4 languages @2019-06-23
+
```
```suggestion
```

Freso commented on taxonomies/other_nutritional_substances.txt:
```diff
@@ -369,7 +372,6 @@ wikidata:en: Q18324709
 # ingredient/choline-citrate has no products @2019-06-23
 
 
```
```suggestion
```

Freso commented on taxonomies/other_nutritional_substances.txt:
```diff
@@ -428,3 +431,4 @@ wikidata:en: Q407997
 # zh-tw:肌醇
 wikipedia:en: https://en.wikipedia.org/wiki/Inositol
 
+
```
```suggestion
```

Freso commented on taxonomies/additives.txt:
```diff
@@ -799,30 +799,30 @@ vegetarian:en: yes
-sv: E126, E126 food additive
+sk: E126, Ponceau 6R
+sl: E126, Ponceau 6R
+sv: E126, Ponceau 6R
```
```suggestion
xx: E126, Ponceau 6R
fr: E126, Ponceau GR, Ponceau 6R
hu: E126, Ponszó 6R
pl: E126, Pąs 6R
```

teolemon committed:
Merge branch 'main' into small-things

teolemon committed:
Apply suggestions from code review

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/other_nutritional_substances.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/unused/categories.wip.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/unused/categories.wip.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/unused/categories.wip.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/unused/categories.wip.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/other_nutritional_substances.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/other_nutritional_substances.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/other_nutritional_substances.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/other_nutritional_substances.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update taxonomies/other_nutritional_substances.txt

Co-authored-by: Freso <freso.dk@gmail.com>

teolemon committed:
Update additives.txt

teolemon committed:
Merge branch 'main' into small-things

teolemon committed:
Merge branch 'main' into small-things

Freso commented on taxonomies/beauty/labels.txt:
```diff
@@ -142,16 +142,35 @@ zh-CN: 不含合成樟脑
+uk: Без змивання
+vi: Không cần rửa
+zh-CN: 无需冲洗
+zh-TW: 無需沖洗
```
AFAIK, we don’t (currently) support locales (or anything, really, that isn’t a two-letter code).
```suggestion
zh: 无需冲洗, 無需沖洗
#zh-CN: 无需冲洗
#zh-TW: 無需沖洗
```

Freso commented on taxonomies/unused/categories.wip.txt:
```diff
@@ -4030,21 +4030,21 @@ fr: Beaumes de Venise
 protected_name_file_number:en: PDO-FR-A0724
 protected_name_type:en: pdo
-#wikidata:en:
+wikidata:en:Q1236939
```
```suggestion
wikidata:en: Q1236939
```

Freso commented on taxonomies/unused/categories.wip.txt:
```diff
@@ -4030,21 +4030,21 @@ fr: Beaumes de Venise
 protected_name_file_number:en: PDO-FR-A0690
 protected_name_type:en: pdo
-#wikidata:en:
+wikidata:en:Q507055
```
```suggestion
wikidata:en: Q507055
```

Freso commented on taxonomies/unused/categories.wip.txt:
```diff
@@ -4030,21 +4030,21 @@ fr: Beaumes de Venise
 protected_name_file_number:en: PDO-FR-A0737
 protected_name_type:en: pdo
-#wikidata:en:
+wikidata:en:Q2922358
```
```suggestion
wikidata:en: Q2922358
```

Freso commented on taxonomies/unused/categories.wip.txt:
```diff
@@ -4219,7 +4219,7 @@ fr: Anjou Villages Brissac
 protected_name_file_number:en: PDO-FR-A0259
 protected_name_type:en: pdo
-#wikidata:en:
+wikidata:en:Q2850261
```
```suggestion
wikidata:en: Q2850261
```

Freso commented on taxonomies/unused/categories.wip.txt:
```diff
@@ -4286,7 +4286,7 @@ fr: Costières de Nîmes
 protected_name_file_number:en: PDO-FR-A0161
 protected_name_type:en: pdo
-#wikidata:en:
+wikidata:en:Q470355
```
```suggestion
wikidata:en: Q470355
```

teolemon replied:
```suggestion
wikidata:en: Q470355
```

Freso reviewed (approved):
Some minor things, but fine to merge regardless, I think.

teolemon committed:
Update taxonomies/beauty/labels.txt

Co-authored-by: Freso <dev@freso.dk>

teolemon committed:
Update taxonomies/unused/categories.wip.txt

Co-authored-by: Freso <dev@freso.dk>

teolemon committed:
Update taxonomies/unused/categories.wip.txt

Co-authored-by: Freso <dev@freso.dk>

teolemon committed:
Update taxonomies/unused/categories.wip.txt

Co-authored-by: Freso <dev@freso.dk>

teolemon committed:
Merge branch 'main' into small-things

teolemon committed:
Apply suggestion from @teolemon

teolemon commented on taxonomies/additives.txt:
```diff
@@ -798,31 +798,10 @@ vegan:en: yes
-es: E126, Ponceau 6R
-et: E126, E126 food additive
-fi: E126, Ponceau 6R
+xx: E126, Ponceau 6R
```
```suggestion
xx: E126, Ponceau 6R
en: E126, Ponceau 6R
```

teolemon committed:
fix: Enhance E125 and E126 entries in additives.txt

Updated entries for food additives E125 and E126 with additional names and translations.
